wee_slack.py

```diff
@@ -2617,6 +2619,45 @@ def process_message(message_json, eventrouter, store=True, **kwargs):
             channel.store_message(message, team)
         dbg("NORMAL REPLY {}".format(message_json))
 
+    if download:
+        download_files(message_json, **kwargs)
+
+
+def download_files(message_json, **kwargs):
+    team = kwargs["team"]
+    download_location = config.files_download_location
+    if not download_location:
+        return
+    if not os.path.exists(download_location):
+        try:
+            os.makedirs(download_location)
+        except:
+            dbg("failed to makedirs({})".format(download_location))
```

Comment:
We should print this to the core buffer (`w.prnt('', 'ERROR: ...')`) instead so it's more apparent that something is failing, and preferably with a bit more descriptive message, e.g. "Failed to create directory at \`files_download_location\`". It should also include the error message from the exception.